ddtrace/contrib/elasticsearch/transport.py

```diff
@@ -41,24 +63,25 @@ def perform_request(self, method, url, params=None, body=None):
                 s.set_tag(metadata.PARAMS, urlencode(params))
                 if method == "GET":
                     s.set_tag(metadata.BODY, self.serializer.dumps(body))
-
                 s = quantize(s)
+                result = super(TracedTransport, self).perform_request(method, url, params=params, body=body)
```

Comment:
for a start, we can wrap this in a try-except, and set a status code on the span. i bet this will give us something close to what we want:

```
from elasticsearch.exceptions import Transport Error
....
try:
    result = super(TracedTransport, self).perform_request(method, url, params=params, body=body)
except TransportError as e:
    s.set_tag(http.STATUS_CODE, e.status_code)
    raise
```